addons/mail/static/src/models/composer_view.js

```diff
@@ -1386,6 +1392,9 @@ registerModel({
             inverse: 'composerViewOwner',
             isCausal: true,
         }),
+        compositions: many('Composition', {
+            inverse: 'composerView',
+        }),
```

Comment:
`sort` by `id`, to avoid relying on arbitrary choice that no specified order here is FIFO due to implementation detail.

If this was LIFO instead, the behaviour would be bad for user, so makes sense to enforce order in modeling.